Entities/Ingame/Club.cs

```diff
@@ -48,13 +48,13 @@ public Club (int memoryAddress, ArraySegment<byte> originalBytes, IVersion versi
 
 		public string Name { 
 			get {
-				return PropertyInvoker.GetString (ClubOffsets.Name, 0, OriginalBytes, MemoryAddress, DatabaseMode);
+				return PropertyInvoker.GetString (ClubOffsets.Name, -1, OriginalBytes, MemoryAddress, DatabaseMode);
 			}
 		}
 
 		public string ShortName {
 			get {
-				return PropertyInvoker.GetString (ClubOffsets.ShortName, 0, OriginalBytes, MemoryAddress, DatabaseMode);
+				return PropertyInvoker.GetString (ClubOffsets.ShortName, -1, OriginalBytes, MemoryAddress, DatabaseMode);
 			}
 		}
 
```

Comment:
Are you 100% sure that's the case in the latest version?  because having addBufferIndex set to 0 for Person.Firstname and Person.Lastname works completely fine and I tried to find Person.Lastname by setting addBufferIndex to 0xC but it can't find the right name.. I tried this with PersonOffsets.Lastname set to every number from 1 to 500.